src/docs/sdk/performance/frames-delay.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: "Frames Delay"
+---
+
+Frames Delay is the user-perceived delayed duration of rendered frames. SDKs can attach this
+information to spans. We recommend implementing this feature for mobile and desktop SDKs.
+
+## Background
+
+This section explains why frames delay is a better metric than slow and frozen frames. If you
+already know why, you can skip this section.
+
+On Mobile, slow and frozen frames are well-established metrics for tracking the responsiveness of
+applications. A phone or tablet typically renders 60 frames per second (fps). At 60 fps, every frame
+has 16 or 16.67 ms to render: 
+
+* __Slow Frames__: Using 60 fps, slow frames are frames that take more than 16 ms (Android) or 16.67 ms
```

Comment:
On Android frame metrics are reported as nano durations, I think we can unify this to `16.67ms`